docs/src/development.md

```diff
@@ -0,0 +1,24 @@
+# Development notes
+
+To run tests:
+```
+julia --project=. -e 'import Pkg; Pkg.test()'
+```
+
+To install dependencies:
+
+```
+julia --project=. -e 'import Pkg; Pkg.instantiate()`
+```
+
+then:
+```
+julia --project=.
```

Comment:
```suggestion
```

Perhaps simply delete these lines? They are ambiguous... "Then" what? The dependencies are already installed from the previous step.